mitodl/open-discussions
# Updated sentry and added filter to exclude typical shutdown errors

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #2578 

#### What's this PR do?
- Upgrades sentry dependencies
- Adds filter to sentry logging to exclude exit errors in python

#### How should this be manually tested?
- Set `SENTRY_DSN` in your `.env` file (you can grab the value from the sentry management page)
- Hardcode an error in python and JS code and hit that code, verify it shows up in sentry
